manifest/syncer.yaml

```diff
@@ -34,6 +34,14 @@ rules:
   - ingresses
   verbs:
   - "*"
+- apiGroups:
+  - "apiextensions.k8s.io"
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - "get"
+  - "watch"
+  - "list"
```

Comment:
We also need apiresourceimports create/update/delete, right?

Reply:
Oh, this is for the pcluster?

Reply:
Yes, this is for the pcluster.